validphys2/src/validphys/fitdata.py

```diff
@@ -187,22 +206,6 @@ def summarise_fits(collected_fit_summaries):
     return pd.concat(collected_fit_summaries, axis=1)
 
 
-def fit_sum_rules(fit, replica_paths):
```

Comment:
If we are removing this (btw should we?), can we remove  `FitSumRulesGrid` as well?

Reply:
I'll check. `fit_sum_rules` was using the `.sumrules` file that we do create but it has been empty since forever. And they get computed every time anyway.